src/block/block_compact.c

```diff
@@ -157,10 +156,8 @@ __wt_block_compact_skip(WT_SESSION_IMPL *session, WT_BLOCK *block, bool *skipp)
     }
 
     __wt_verbose_debug(session, WT_VERB_COMPACT,
-      "%s: total reviewed %" PRIu64 " pages, total skipped %" PRIu64 " pages, total wrote %" PRIu64
-      " pages",
-      block->name, block->compact_pages_reviewed, block->compact_pages_skipped,
-      block->compact_pages_rewritten);
+      "%s: total reviewed %" PRIu64 " pages, total wrote %" PRIu64 " pages", block->name,
```

Comment:
As long as you're changing this, would you also change "wrote" to "rewritten" to match the language of the other message.

Reply:
Yes can do! Good point.